### PR TITLE
doc: add minutes for meeting Oct 24 2018

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2018-11-24.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+### nodejs/node
+
+* buffer: fix crash for invalid index types [#23795](https://github.com/nodejs/node/pull/23795)
+  * removed from the agenda before the meeting
+  
+* fs: behaviour of readFile and writeFile with file descriptors [23433]( https://github.com/nodejs/node/issues/23433)
+  * Behaviour is inconsistent with the documentation.
+  * Read and Write behave differently
+  * PR exists to update documentation
+  * Call for people to get involved in the issue.
```
Comment (thefourtheye) [inline meetings/2018-11-24.md]: You have summarized this problem perfectly. 👏

File: meetings/2018-11-24.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
+  * No Myles, skipped for this week
+
+
+
+
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+  * N-API
+    * Getting ready for workshopp at NodeConfEU
```
Comment (thefourtheye) [inline meetings/2018-11-24.md]: Typo: workshop

File: meetings/2018-11-24.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  * Read and Write behave differently
+  * PR exists to update documentation
+  * Call for people to get involved in the issue.
+
+### nodejs/TSC
+
+* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
+  * No Myles, skipped for this week
+
+
```
Comment (thefourtheye) [inline meetings/2018-11-24.md]: Nit: We don't need these many empty lines, right?

**Comment by cjihrig** [comment]
The filename seems wrong to me: 2018-11-24.md (should be 2018-10-24).

File: meetings/2018-11-24.md (hunk 3 above)
Comment (mhdawson) [reply meetings/2018-11-24.md]: Looked in the file, not sure while more spaces are shown as there was only one.  Tried deleting/re-adding a single blank line.

Comment (mhdawson) [reply meetings/2018-11-24.md]: Ah was looking at version without one of the commits.

**Comment by mhdawson** [comment]
@cjihrig good catch, fixed.

**Pull request merged**